### ISSUE TITLE
508-defect-4 [COGNITION]: App Directory - Additional information lists should be unordered lists

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#16466]
# [508-defect-4](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-4)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## User Story or Problem Statement

As a screen reader user, I generally expect ordered lists to give me steps in a particular order. These lists appear to be unordered options for things apps can provide me.

## Details

The additional information lists have type: disc, but they're ordered lists. These would be better served as unordered lists because they're not steps that need to be followed in an order. Screenshot attached below.

## Acceptance Criteria

- [ ] Additional information lists are changed to UL from OL
- [ ] No axe violations

## WCAG or Vendor Guidance (optional)

* [Info and Relationships: Understanding SC 1.3.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html)

## Screenshots or Trace Logs

![Screen Shot 2020-11-19 at 8 42 22 PM](https://user-images.githubusercontent.com/934879/99751470-de27fe00-2aa7-11eb-8dad-a091a338690b.png)

**Comment by LeKeve** [comment]
Ready to test in production.

**Comment by 1Copenut** [comment]
LGTM. Moving to close.
